toolkit/tools/scheduler/schedulerutils/buildlist.go

```diff
@@ -123,3 +126,16 @@ func ReadReservedFilesList(path string) (reservedFiles []string, err error) {
 
 	return reservedFiles, nil
 }
+
+// IsToolchainRPM determines if a given RPM path is found in a list of reserved RPMs.
+// rpmsToFind should be an absolute path to the expected RPM, while toolchainRPMs is a list of '<name>-<version>-<release>.rpm'.
+func IsToolchainRPM(rpmPath string, reservedRPMs []string) bool {
```

Comment:
Seeing how the caller must pass the list of reserved RPMs anyway, I'd re-name it to something more generic like `IsReservedRPM`. Or even `IsReservedFile`. The caller can still give more context about the returned value with a correct choice of a variable name.

Reply:
Done